### ISSUE TITLE
Extent of annotation list is hard to follow

### ISSUE THREAD

**Issue by ceesem** [seung-lab/neuroglancer#508]
When working with lots of annotations, for example a list of locations to proofread, one often wants to be able to clearly track progress through a long list of annotations. However, annotations are displayed in a panel that only can display ~15 point annotations, and there is no scroll bar and other human-readable identifying line information (e.g. annotation number). This makes it extremely hard to keep track of specific annotations, or to follow ones progress through a list.

**Comment by jefferis** [comment]
We have been thinking about using annotation shortcuts (Shift+Q etc) to maintain status when reviewing but it doesn't seem ideal.

**Comment by ceesem** [comment]
Yeah, that's exactly what I do. I also try to created reasonably grouped annotations (e.g. checking all branch points down a particular branch) and only use the shortcut to mark, e.g., the first annotation in the group just to avoid hitting shift-Q hundreds of times.
